### PR TITLE
Devboris02 pre 13 setup unit test in the project

### PR DESCRIPTION
**Pull request by sonwaroberto**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ACCESS_TOKEN = "yIR9I7shhcWPnH6AUlvmb7tryAfgNDEHI3GjxUA6uWs"
```
Comment (Nanfack-Takoutsi-Yvan) [inline .env]: this is BAD, how could `.env` files be found in a commit? It was supposed to be ignored.

**Review by Nanfack-Takoutsi-Yvan — COMMENTED** [review]
I can't accept this commit because:

1. You went out of the scope of the task
2. we can find `.env` file in the commit and add key samples
3. Your commits are unsigned
4. Your commit messages don't have a scope and don't follow [conventional commit rules](https://www.conventionalcommits.org/en/v1.0.0/)
5. The pr seems too big

**Comment by Nanfack-Takoutsi-Yvan** [comment]
> 

I want this project to follow all good practices

File: .env (hunk 1 above)
Comment (sonwaroberto) [reply .env]: alright boss...

**Comment by sonwaroberto** [comment]
alright i'll go && charge my pc, work on it again during the night
so i should reduce the tasks on the pr before push it again ??

**Pull request closed without merging**